json-definitions/v3/tech-record/put/hgv/complete/index.json

```diff
@@ -224,10 +224,10 @@
     },
     "techRecord_adrDetails_weight": {
       "type": [
-        "string",
+        "number",
         "null"
       ],
-      "maxLength": 8
+      "maxLength": 99999999
```

Comment:
this looks wrong

Reply:
it is it should be max, I'll sort it now

Reply:
https://json-schema.org/understanding-json-schema/reference/numeric#range it uses maximum